test/e2e/elasticache/replication_group/coverage_tests/rg_misc.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
```

Comment:
same comment about the file name... maybe just `test/e2e/elasticache/replication_group/misc.sh`?

Reply:
yep I'll remove the redundant prefix